apps/wakunode2/config.nim

```diff
@@ -63,25 +67,43 @@ type
       defaultValue: 50
       name: "max-connections" }: uint16
     
+    peersPersistence* {.
```

Comment:
What do you think about making the prefix singular `peer-`? That way it can be applied as a general category for everything `peer` related. At least in schema design (e.g. table names) singular is usually preferred, though I know people have different ideas here. :)